active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,324 @@
+| PipsPagerIndicatorWidth | Sets the bounding box width for each pip glyph | Double
+| PipsPagerIndicatorHeight | Sets the bounding box height for each pip glyph | Double
+
+
+## PipsPager member notes
+
+| Name | Description| Default | 
+|:---:|:---|:---|
+| NumberOfPages | Sets the max number of pages the index control will iterate through. The default will represent an infinite page range. | -1
+| Previous and Next Style | Give the developer the option to customize the style by changing the text or glyph for the edge buttons. | N/A
+| SelectedPageIndex | The 0 based index that is currently selected. It will default to the first index. | 0
+| MaxVisiblePips | The maximum number of pips that will appear in the control at once. If the NumberOfPages is greater than this value, the UI will scroll so that the selected pip is centered. | 5
+| Orientation | The orientation of the control. Can be either Vertical or Horizontal. | Horizontal
+| PipsPagerSelectedIndexChangedEvent | Event that is fired after the user selects a pip or the directional buttons. This event will return the index number of the page that the user selected. | N/A
+
+## PipsPagerButtonVisibility enum
+
+Defines constants that specify how a the buttons of a PipsPager are to be displayed. Defaults to Collapsed.
+
+| Value | Description |
+|:---:|:---|
+| Visible | The button is visible and selectable. An exception to this is that if the selected page is on the associated "edge" of the button then the button will be hidden |
+| VisibleOnPointerOver | The behavior is the same as Visible *except* that the button is only Visible when the user is hovering over the paging UI with their cursor. |
+| Collapsed | The button is not visible to the user and does **not** take up layout space. |
+
+
+# API Details
+
+```IDL
+enum PipsPagerButtonVisibility
+{
+  Visible,
+  VisibleOnPointerOver,
+  Collapsed, 
+}
+
+runtimeclass PipsPagerSelectedIndexChangedEventArgs
+{
+    Int32 NewPageIndex{get; };
+    Int32 PreviousPageIndex{get; };
+};
+
+unsealed runtimeclass PipsPagerTemplateSettings : Windows.UI.Xaml.DependencyObject
+{
+    Windows.Foundation.Collections.IVector<Object> PipsPagerItems { get; }; 
+}
+
+
+unsealed runtimeclass PipsPager : Windows.UI.Xaml.Controls.Control
+{
+    PipsPager();
+
+    [MUX_DEFAULT_VALUE("-1")]
+    Int32 NumberOfPages;
+
+    [MUX_DEFAULT_VALUE("0")]
+    Int32 SelectedPageIndex;
+
+    [MUX_DEFAULT_VALUE("5")]
+    Int32 MaxVisiblePips;
+
+    [MUX_DEFAULT_VALUE("Horizontal")]
+    Windows.UI.Xaml.Controls.Orientation Orientation;
+
+    [MUX_DEFAULT_VALUE("Collapsed")]
+    PagerControlButtonVisibility PreviousButtonVisibility;    
+    [MUX_DEFAULT_VALUE("Collapsed")]
+    PagerControlButtonVisibility NextButtonVisibility;
+
+    Windows.UI.Xaml.Style PreviousButtonStyle;
+    Windows.UI.Xaml.Style NextButtonStyle; 
+
+    Windows.UI.Xaml.Style SelectedPipStyle;
+    Windows.UI.Xaml.Style DefaultPipStyle;
```

Comment:
Suggest (minor) calling this NormalPipStyle. I can't find API precedent, but what this represents is what Visual State Groups usually call the Normal state.

Reply:
I'll leave this unresolved until extended review, thanks!

Reply:
Recopmmend: NormalPipStyle